apps/meteor/app/api/server/v1/channels.ts

```diff
@@ -70,7 +70,7 @@ async function findChannelByIdOrName({
 		room = await Rooms.findOneByName(params.roomName || '', { projection });
 	}
 
-	if (!room || (room.t !== 'c' && room.t !== 'l')) {
+	if (!room || !['c', 'l', 'v'].includes(room.t)) {
```

Comment:
We should not add a new value here (`v`), previously all the `channels.` endpoints were not accepting `v` as a valid value, and after this change, they will.

Can you explain why do we need to add that extra value there?

Also, if this will be used only for `.open` endpoint, it would be better to add that check there only. Otherwise we need to add tests for all the endpoints

Reply:
> Can you explain why do we need to add that extra value there?

Looking at the other `open` endpoints we currently have, none seem to accept `v` rooms. 
Currently we allow `v` rooms to be hidden by calling `channels.close`, it seemed logical to use `channels.open` to open the same type of room.

Initially my plan was to add a new endpoint with similar restrictions to the meteor method being replaced, but considering that we have the `.open` endpoints I was wondering if there could be a better solution using what we already have.